app/assets/javascripts/practice_editor_adoptions.es6

```diff
@@ -27,6 +27,11 @@
         event.preventDefault();
         const formId = $(event.target).data('form-id');
         const formIdSelector = `#${formId}`;
+
+        // clear all alerts
```

Comment:
could we also clear the alert in the  `clearAdoptionEntryFrom` on click as well? Thank you!!